projects/dsp-ui/src/lib/search/expert-search/expert-search.component.ts

```diff
@@ -123,4 +125,16 @@ CONSTRUCT {
         return query + offsetTemplate;
     }
 
+    /**
+     * Get the IRI base url without configured api protocol.
+     * The protocol in this case is always http
+     */
+    getIriBaseUrl(): string {
```

Comment:
It is not used on view, so could be marked as `private`.

Reply:
Done in 014cc13